### PR TITLE
Resolved Support Version 2.0.0 changes for regard-application

### LINKED ISSUE #206: Support Version 2.0.0 changes for regard-application
1. Generate server stub for the version 2.0.0 specification 
2. If the api/openapi.yaml is not updated as per the published version 2.0.0 , then copy the api/openapi.yaml file from the generated stub to the source code
3. no change required in the method signature

Following tasks needs to be completed to support regard-application version 2.0.0 , 

- [x] https://github.com/openBackhaul/ApplicationLayerTopology/issues/202
- [x] https://github.com/openBackhaul/ApplicationLayerTopology/issues/203
- [x] https://github.com/openBackhaul/ApplicationLayerTopology/issues/204
- [x] https://github.com/openBackhaul/ApplicationLayerTopology/issues/205
- [x] https://github.com/openBackhaul/ApplicationLayerTopology/issues/193

### PR DESCRIPTION
**Pull request by ManasaBM1**
Fixes #206

### CODE REVIEW

**Comment by ManasaBM1** [comment]
Reviewer is @vanithavalluripalli9

File: server/service/individualServices/IndividualServicesOperationsMapping.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports.individualServicesOperationsMapping =
```
Comment (DanaSunal) [inline server/service/individualServices/IndividualServicesOperationsMapping.js:1]: Mapping should be just one file. Take a look at BasicServicesOperationsMapping in ApplicationPattern.

**Review by DanaSunal — CHANGES_REQUESTED** [review]
Mappings should be in one file, not a file per method.

File: server/service/individualServices/IndividualServicesOperationsMapping.js (hunk 1 above)
Comment (ManasaBM1) [reply server/service/individualServices/IndividualServicesOperationsMapping.js:1]: ok done

File: server/service/IndividualServicesService.js [hunk 2]
```diff
@@ -7,13 +8,18 @@ const layerProtocol = require('../applicationPattern/onfModel/models/LayerProtoc
...
+const individualServicesOperationsMapping = require('./individualServices/IndividualServicesOperationsMapping');
 const ForwardingConfigurationService = require('onf-core-model-ap/applicationPattern/onfModel/services/ForwardingConstructConfigurationServices');
 const ForwardingAutomationService = require('onf-core-model-ap/applicationPattern/onfModel/services/ForwardingConstructAutomationServices');
 
 const FcPort = require("onf-core-model-ap/applicationPattern/onfModel/models/FcPort"); 
 
 const prepareForwardingConfiguration = require('./individualServices/PrepareForwardingConfiguration');
 const prepareForwardingAutomation = require('./individualServices/PrepareForwardingAutomation');
+
+const httpServerInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/HttpServerInterface');
```
Comment (DanaSunal) [inline server/service/IndividualServicesService.js:20]: Already on line 27

File: server/service/IndividualServicesService.js [hunk 3]
```diff
@@ -7,13 +8,18 @@ const layerProtocol = require('../applicationPattern/onfModel/models/LayerProtoc
...
+const tcpServerInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/TcpServerInterface');
+
```
Comment (DanaSunal) [inline server/service/IndividualServicesService.js:22]: Already on line 30

File: server/service/individualServices/IndividualServicesOperationsMapping.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+          }
+      },
+      "/v1/regard-application": 
+      {
+        "redirect-topology-change-information": {
+            "api-segment": "im",
+            "sequence": "005"
+          }
+      }
+}
```
Comment (DanaSunal) [inline server/service/individualServices/IndividualServicesOperationsMapping.js:17]: Format this json.

**Review by DanaSunal — CHANGES_REQUESTED** [review]

File: server/service/IndividualServicesService.js (hunk 2 above)
Comment (ManasaBM1) [reply server/service/IndividualServicesService.js]: removed

File: server/service/individualServices/IndividualServicesOperationsMapping.js (hunk 4 above)
Comment (ManasaBM1) [reply server/service/individualServices/IndividualServicesOperationsMapping.js:15]: done

**Review by DanaSunal — APPROVED** [review]

**Pull request merged**